openspec/changes/2026-01-30-add-qwen-support-and-session-management/proposal.md

```
@@ -0,0 +1,64 @@
+# Add Qwen ACP Support and Enhanced Session Management
+
+## Summary
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)